repose-aggregator/components/filters/keystone-v2/src/main/scala/org/openrepose/filters/keystonev2/KeystoneV2Filter.scala

```diff
@@ -675,4 +677,6 @@ object KeystoneV2Filter {
 
   case class InvalidTenantException(message: String, cause: Throwable = null) extends Exception(message, cause)
 
+  object Pass extends KeystoneV2Result
+
```

Comment:
the auto formatter seems to be overly aggressive on scala code, i'm leaning towards trusting it on spacing, but not placement